data-plane/examples/src/docker/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+# filepath: /Users/amitami2/personal/AI/IOA/slim/data-plane/examples/src/docker/docker-compose.yml
+# Copyright AGNTCY Contributors (https://github.com/agntcy)
+# SPDX-License-Identifier: Apache-2.0
+
+version: '3.8'
+
+services:
+  # Slim server
+  slim-server:
+    build:
+      context: ../../../../
+      dockerfile: ./data-plane/examples/src/docker/Dockerfile.slim
+    ports:
+      - "50051:50051"
+    networks:
+      - slim-network
+    volumes:
+      - ./config/docker-server-config.yaml:/config/server-config.yaml
+    command: ["--config", "/config/server-config.yaml"]
+
+  # Mock agent server
+  mock-agent-server:
+    build:
+      context: ../../../../
+      dockerfile: ./data-plane/examples/src/docker/Dockerfile.mock-agent-server
```

Comment:
Add the target here to specify the target in the Dockerfile (mock-agent-server).